content/docs/csidriver/installation/operator/_index.md

```diff
@@ -263,6 +296,7 @@ The CSI Drivers installed by the Dell CSI Operator can be updated like any Kuber
     ```
     and modify the installation
 * Modify the API object in-place via `kubectl patch`
+* Modify the API object via `kubectl apply -f <file>` . Refer [here](https://github.com/dell/dell-csi-operator/tree/master/samples) to create a CR (Custom Resource) using the sample files provided.
```

Comment:
Please remove the above line as we have the below mentioned **NOTE** in the same file 
NOTE: Do not try to update the operator by modifying the original CustomResource manifest file and running the kubectl apply -f command. As part of the driver installation, the Operator sets some annotations on the CustomResource object which are further utilized in some workflows (like detecting upgrade of drivers). If you run the kubectl apply -f command to update the driver, these annotations are overwritten and this may lead to failures.

Reply:
Removed